src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -371,6 +371,11 @@ void reshape_to_static(std::shared_ptr<ov::Model> model,
         ov::PartialShape new_shape;
         if (input_name.find("input_ids") != std::string::npos) {
             new_shape = ov::PartialShape({1, input_size});
+        } else if (input_name.find("inputs_embeds") != std::string::npos) {
+            // NB: VLMs case, model accepts inputs_embeds[BATCH, SEQ_LEN, EMB_SIZE]
+            OPENVINO_ASSERT(input.get_partial_shape().size() == 3u);
+            OPENVINO_ASSERT(input.get_partial_shape()[2].is_static());
```

Comment:
I thought we've had NPUW_ASSERT or smt like that

Reply:
You're right, replaced